charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -10,36 +10,37 @@ data:
   cilium: |
     ChartName: cilium
     ChartVersion: 1.15.0
-    RepositoryURL: oci://mindthegap.{{ .Release.Namespace }}.svc
+    RepositoryURL: oci://mindthegap.caren-system.svc/charts
```

Comment:
This won't work if deployed in a different namespace?

Reply:
that is correct. it needs to be deployed in that namespace, this is a limitation of trust-manager needing trust namespace set. this comment explains it 

https://github.com/cert-manager/trust-manager/issues/120#issuecomment-1549984274

if helm was able to use templating in values.yaml we could get around this